cpp/convert_json_to_marc.cc

```diff
@@ -58,9 +58,10 @@ namespace {
 struct JournalTitleAndPPN {
     std::string journal_title_;
     std::string ppn_;
+    std::string online_issn_;
```

Comment:
either rename struct from JournalTitleAndPPN to JournalTitleAndPPNAndOnlineISSN or, before space & nerves run out, to something scalable/generic like JournalInfo